doc/man/cockpit.conf.xml

```diff
@@ -38,11 +38,14 @@
   <refsect1 id="cockpit-conf-description">
     <title>DESCRIPTION</title>
   <para>
-    Cockpit can be configured via /etc/cockpit/cockpit.conf.  This file is not required
-    and may need to be created manually.  The file has a INI file syntax and thus
-    contains key / value pairs, grouped into topical groups. See the examples below for
-    details..
-  </para>
+    Cockpit can be configured via /etc/cockpit/cockpit.conf. If <code>$XDG_CONFIG_DIRS</code>
+    is set, then the first path containing a <filename>../cockpit/cockpit.conf</filename> is used
+    instead. The same applies to all other cockpit related configuration directories.</para>
```

Comment:
"The same applies to" is potentially ambiguous.  The same rules for finding the directory, or the same directory that we found with `cockpit.conf` inside of it?

Reply:
"The same search rule applies.." perhaps?